### ISSUE TITLE
Make mobile friendly version?

### ISSUE THREAD

**Issue by iangilman** [openseadragon/site-build#226]
The OpenSeadragon site is kind of anachronistic at this point, with its desktop-only presentation. OpenSeadragon itself supports mobile just fine, so maybe our site should as well?

In particular, if you come to our site trying to see how OpenSeadragon performs on mobile, you might be disappointed, because it doesn't perform as well on our scaled-down desktop site as it would on a regular mobile site.

On the other hand, it's an informational site for developers, so people are most likely going to be visiting it on desktop. Our search stats from last month bear this out:

<img width="259" alt="image" src="https://user-images.githubusercontent.com/338989/90049492-ba277600-dc89-11ea-8cba-f6ba2ef8752d.png">

At any rate, making the whole site mobile-friendly would be a big production. It would perhaps be worth doing an entire site redesign if we're going to do it. I don't see it happening in the near future, but I thought I'd start this thread so we can at least muse about it.

**Comment by msalsbery** [comment]
Just mentioning this because it’s my go-to stack currently...

React with react-bootstrap makes responsive design a no-brainer for even a non-frontend dev like me ;)

**Comment by msalsbery** [comment]
Love the idea though!

What I dread is doing the same to the jsdoc-generated documentation as well. There may be some more modern template boilerplate around online however...

**Comment by msalsbery** [comment]
> React with react-bootstrap

Ha! Not one comment ;) React not really appropriate for the site, but I do find Bootstrap makes responsive design pretty easy...

**Comment by iangilman** [comment]
😄 I dunno… I'm pretty agnostic about how it's put together (though I wouldn't want it to end up looking like a generic Bootstrap site). React is probably more trouble than it's worth, since it's not really a dynamic site, so you'd be adding a bunch of extra code to your HTML for no reason.

I figure if we're going to do a site redesign it's going to be a good deal of work, so I want it to be:

* Pleasant for whoever's doing it
* A nice improvement for our users
* No harder to maintain than the current site

Beyond that I'm pretty flexible 🙂

**Comment by msalsbery** [comment]
@iangilman 

Yeah, the more I think about it, even Bootstrap is overkill. What I like about Bootstrap goes with working with React - for SPAs. When I look at the CSS that just applies to responsive design, it’s easier to use straight CSS
